docs/publish-docs.sh

```diff
@@ -2,153 +2,186 @@
 
 set -e
 
-if [[ -d .vercel ]]; then
-  rm -r .vercel
-fi
+here=$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")" &>/dev/null && pwd)
 
-CONFIG_FILE=vercel.json
+# shellcheck source=ci/env.sh
+source "$here/../ci/env.sh"
 
-if [[ -n $CI_TAG ]]; then
-  PROJECT_NAME=docs-anza-xyz
+if [[ -z $CI ]]; then
+  echo "Publishing docs from local"
+  if [[ -z $PROJECT_NAME ]]; then
+    echo "❌ PROJECT_NAME is undefined"
+    exit 1
+  fi
+  PROJECT_NAMES=("$PROJECT_NAME")
 else
-  eval "$(../ci/channel-info.sh)"
-  case $CHANNEL in
-  edge)
-    PROJECT_NAME=edge-docs-anza-xyz
-    ;;
-  beta)
-    PROJECT_NAME=beta-docs-anza-xyz
-    ;;
-  *)
-    PROJECT_NAME=docs
-    ;;
-  esac
+  echo "Publishing docs from CI"
+
+  # skip docs publish from pull requests
+  if [[ -n $CI_PULL_REQUEST ]]; then
+    echo "skipping docs publish from pull requests"
+    exit 0
+  fi
+
+  # get the channel info
+  eval "$("$here/../ci/channel-info.sh")"
+  if [[ -n $CI_TAG ]]; then
+    if [[ $CI_TAG != $BETA_CHANNEL* ]]; then
+      echo "skipping docs publish from non-beta tag"
+      exit 0
+    fi
+    PROJECT_NAMES=("docs-anza-xyz" "beta-docs-anza-xyz")
```

Comment:
push to 2 docs projects if it's a beta tag

Reply:
I can't remember why it isn't setup this way... maybe we were planning a DNS redirect for beta.

But doing it explicitly sounds good to me